rules/upgrade-dnn-to-the-latest-version/rule.md

```diff
@@ -0,0 +1,55 @@
+---
+type: rule
+title: Do you know how to upgrade DNN to the latest version?
+uri: upgrade-dnn-to-the-latest-version
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+created: 2014-03-14T03:24:00.000Z
+archivedreason: outdated
+guid: 08e851b5-52a1-4fac-a73a-7f20307c80f5
+---
+If there is an upgrade available (see ["Have you installed the latest DNN version?"](https://ssw.com.au/ssw/Standards/Rules/RulestoBetterDotNetNuke.aspx#LatestVersion) ), follow the following steps to perform the upgrade.
+
+<!--endintro-->
+
+**In Preparation:**
+
+1. Back up the DNN database
+2. Back up all website files
+3. Check your database and website backups!
+4. Download the latest DotNetNuke upgrade package:
+
+   1. From the www.dotnetnuke.com site. You may need login credentials if you are using a non-community version of DNN or
+   2. Click the "Upgrade Available" button that appears when you are *not* running the latest version (see ["Have you installed the latest DNN version?"](https://ssw.com.au/ssw/Standards/Rules/RulestoBetterDotNetNuke.aspx#LatestVersion)).
```

Comment:
Hey @fenix2222 

This link should be updated since it's all migrated

- [ ] 1. Please action

Reply:
Done for @fenix2222 to get this merged